odl/test/tomo/backends/astra_setup_test.py

```diff
@@ -154,23 +153,6 @@ def test_vol_geom_3d():
             odl.tomo.astra_volume_geometry(discr_dom)
 
 
-def test_proj_geom_parallel_2d():
-    """Create ASTRA 2D projection geometry."""
-
-    apart = odl.uniform_partition(0, 2, 5)
-    dpart = odl.uniform_partition(-1, 1, 10)
-    geom = odl.tomo.Parallel2dGeometry(apart, dpart)
-
-    proj_geom = odl.tomo.astra_projection_geometry(geom)
-
-    correct_subdict = {
-        'type': 'parallel',
-        'DetectorCount': 10, 'DetectorWidth': 0.2}
-
-    assert is_subdict(correct_subdict, proj_geom)
-    assert 'ProjectionAngles' in proj_geom
```

Comment:
This test was kind of pointless since it really tests ASTRA rather than ODL, and some implementation detail of ODL. Not sure if we really need a replacement.